js/webtest_js.mli

```diff
@@ -7,4 +7,6 @@ val setup : Webtest.Suite.t -> unit
 (** [setup test] sets up a test runner and attaches it to the document's onLoad
     handler. *)
 
+val run : ?with_colors:bool -> Webtest.Suite.t -> unit
```

Comment:
A doc comment explaining how to use this would be helpful. I assume this is the toplevel function to call when running tests under node.js?